Enterprise-CMCS/macpro-onemac
# Oy2 24002

Story: https://qmacbis.atlassian.net/browse/OY2-24002
Endpoint: See github-actions bot comment

### Details

Rework how email addresses operate such that any function can override the ccEmail property in serverless config without having to add a new structure to form config

### Changes

- Made default form config look at process.env.ccEmail rather than only override on certain forms
- Made chip special emails for reviewer and cc use standard reviewerEmail and ccEmail rather than different env var names
- updated form configs to follow the above approach
- added new ssm parameter store properties for the 3 new email groups
- updated serverless config to set proper email groups per method for ccEmail

### Implementation Notes

- As these changes rewire how env vars are set along with adding new ssm parameters; the values will need to be manually set after release to prod

### Test Plan

1. Verify each form type still submits
2. Verify setting reviewerEmail and ccEmail on each of the affected form types sends to proper email address
